Cargo.toml

```
@@ -8,7 +8,7 @@ edition = "2018"
 anyhow = "1"
 atty = "0.2"
 futures = { version = "0.3", default-features = false }
-libp2p = { git = "https://github.com/comit-network/rust-libp2p.git", branch = "rendezvous", default-features = false, features = [ "rendezvous", "tcp-tokio", "yamux", "mplex", "dns-tokio", "noise", "ping", "websocket" ] }
+libp2p = { git = "https://github.com/libp2p/rust-libp2p.git", default-features = false, features = [ "tcp-tokio", "yamux", "mplex", "dns-tokio", "noise", "request-response", "websocket", "ping", "rendezvous" ] }
```

I think this slipped in by accident.

```suggestion
libp2p = { git = "https://github.com/libp2p/rust-libp2p.git", default-features = false, features = [ "tcp-tokio", "yamux", "mplex", "dns-tokio", "noise", "websocket", "ping", "rendezvous" ] }
```

---

I don't quite understand why the features changed at all actually. Can you just replace the URL please?

---

i am also confused, I dont remember touching this